.github/workflows/e2e-unmanaged-cluster.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: E2E Test - Unamanged Cluster
+
+on:
+  repository_dispatch:
+    types: [daily-build]
+
+  pull_request_target:
+    types: [opened, reopened, synchronize, ready_for_review]
+    paths:
+      - cli/cmd/plugin/unmanaged-cluster
+
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - release-*
+
+    paths:
+      - "cli/cmd/plugin/unmanaged-cluster/*.go"
+      - "cli/cmd/plugin/"
+      - "**.sh"
+      - ".github/workflows/e2e-unmanaged-cluster.yaml"
+      - "**.md"
```

Comment:
Do we really want to run e2e tests on markdown file changes?

Reply:
Yeah we can avoid this. 
Thanks!